npm-debug.log

```diff
@@ -0,0 +1,20 @@
+0 info it worked if it ends with ok
```

Comment:
uh oh!

(I'd recommend removing this / adding `npm-debug.log` to your global gitignore